components/module-cellery/cellery-utils/src/main/java/io/cellery/CelleryUtils.java

```diff
@@ -981,9 +981,21 @@ public static String getBallerinaExecutablePath() {
                 }, environment,
                 "ballerina", "version");
         if (balVersionCmdOutput.contains("Ballerina")) {
-            String ballerinaVersion = balVersionCmdOutput.split(" ")[1];
-            if (ballerinaVersion.equals(CelleryConstants.BALLERINA_VERSION)) {
-                return "";
+            if (balVersionCmdOutput.split("Language").length > 0) {
+                // Ballerina version > 1.0.0
+                String balVersionOutput = balVersionCmdOutput.split("Language")[0];
+                if (balVersionOutput.split(" ").length > 0) {
+                    String ballerinaVersion = balVersionOutput.split(" ")[1];
+                    if (ballerinaVersion.equals(CelleryConstants.BALLERINA_VERSION)) {
+                        return "";
+                    }
+                }
+            } else {
```

Comment:
Do we need this now ? Since we won't go back below < 1.0.0 again

Reply:
Updated.